<ui version="4.0">
 <class>mapgeoDialogBase</class>
 <widget class="QDialog" name="mapgeoDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mapeamento Geologico</string>
  </property>

  <!-- Botão para Selecionar Geometria -->
  <widget class="QPushButton" name="selectGeometryButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>150</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Selecionar Geometria</string>
   </property>
  </widget>

  <!-- Combo Box para Seleção de Escala -->
  <widget class="QComboBox" name="scaleComboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>150</width>
     <height>32</height>
    </rect>
   </property>
  </widget>

  <!-- Botão de Execução -->
  <widget class="QPushButton" name="runButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>150</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Executar</string>
   </property>
  </widget>

  <!-- Botões de OK e Cancelar -->
  <widget class="QDialogButtonBox" name="button_box">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
